nodes/core/core/89-trigger.js

```diff
@@ -80,7 +80,7 @@ module.exports = function(RED) {
             var topic = msg.topic || "_none";
             if (node.bytopic === "all") { topic = "_none"; }
             node.topics[topic] = node.topics[topic] || {};
-            if (msg.hasOwnProperty("reset") || ((node.reset !== '') && (msg.payload == node.reset)) ) {
+            if (msg.hasOwnProperty("reset") || ((node.reset !== '') && msg.payload && msg.payload.toString && (msg.payload.toString() == node.reset)) ) {
```

Comment:
This won't work if the desired reset word is `"false"` - as `msg.payload` will be `false` and this check won't pass.

Reply:
thanks